.travis.yml

```diff
@@ -4,11 +4,12 @@ notifications:
   email: false
 
 elixir:
-  - 1.6
+  - 1.7
+  - 1.8
```

Comment:
We need to add 1.9 now, right?

Reply:
@doomspork is there a version `1.9` released? I wasn't aware of it.

Reply:
I'm confused, you're right 👍